features/build/stibuild.feature

```diff
@@ -124,3 +124,14 @@ Feature: stibuild.feature
     Then the step should succeed
     And the output should contain "error"
 
+  # @author wzheng@redhat.com
+  # @case_id OCP-22596
+  Scenario: Create app with template eap72-basic-s2i with jbosseap rhel7 image
+    Given I have a project
+    When I run the :new_app client command with:
+      | template | eap72-basic-s2i |
+    Then the step should succeed
+    Given the "eap-app-1" build was created
```

Comment:
@wzheng1 is the `created` status check absolutely needed?  I find that intermediate status check to be hard to catch and could be a source of un-reliability for the test.

Reply:
If the needed imagestream exits under openshift imagestream, the build should be created soon.